projects/angular/src/forms/common/if-control-state/if-error.ts

```diff
@@ -35,27 +35,30 @@ export class ClrIfError extends AbstractIfState {
    * @param state CONTROL_STATE
    */
   protected override handleState(state: CONTROL_STATE) {
-    if (this.error && this.control) {
+    if (this.error && this.control && !!this.control.invalid) {
```

Comment:
```suggestion
    if (this.error && this.control && this.control.invalid) {
```

No need for type coerce here. The `invalid` property is a `boolean`.